backend/typescript/rest/campRoutes.ts

```diff
@@ -31,15 +32,22 @@ const campService: ICampService = new CampService(fileStorageService);
 
 // ROLES: Admin + CC
```

Comment:
Shouldn't parents be able to browse all camps?

Reply:
@dhmi19 ah oops forgot, no camp browsing right?

Reply:
yeah no browsing. should be protected to cc and admin only